src/electionguard/voting/encrypter.c

```diff
@@ -266,7 +269,8 @@ Voting_Encrypter_encrypt_ballot(Voting_Encrypter encrypter,
             tally, encrypter->base_hash, encrypter->joint_key.public_key);
```

Comment:
the selected count should be configured per-contest and all of the ballots with that contest should have exactly the same number of selections